src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/VSCodeAnalyzerLoader.cs

```diff
@@ -51,5 +58,8 @@ public Assembly LoadFromPath(string fullPath)
 
             return defaultLoader.LoadFromPath(fullPath);
         }
+
+        public void UnloadAll()
+            => defaultLoader.UnloadAll();
```

Comment:
When `UnloadAll` is called do we create a new instance of this type? Keeping the existing one around likely has some issues because it's caching items like `AssemblyName` that you want to be different when you're live reloading.

Reply:
> When UnloadAll is called

We only call it in the finalizer of the type that owns this AnalyzerLoader.  So nothing is holding onto it anymore.  

> do we create a new instance of this type?

Morally 'yes'.  We will have thrown one away because we moved to a new one.